### PR TITLE
Generic r script

### PR DESCRIPTION
**Pull request by cfelletter**
Pull into master

### CODE REVIEW

File: code/PostProcess&PlotScript_Daily.R [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+#scenarios you want to compare 
+scens <- list( #Short Name for Run = File Name with RDFs
+  "April 2018" = "Apr2018,DNF,2007Dems,IG,MTOM_Most", 
+  "Asp.NoTribs" = "9601.NoTribs,DNF,2007Dems,IG.9003.NoTribs,MTOM_Most"
+)
+
+list.files(file.path(iFolder,scens[1])) #list files in scen folder for next input
+
+#rdf file with slot you want 
+file = "DailyFlows.rdf"
```
Comment (rabutler-usbr) [inline code/PostProcess&PlotScript_Daily.R]: starting here, many of the variables should use `<-` instead of `=`

**Comment by cfelletter** [comment]
@rabutler-usbr changes made

**Comment by rabutler-usbr** [comment]
Thanks for addressing my annoying requests @cfelletter

**Comment by rabutler-usbr** [comment]
@cfelletter I won't delete the generic_R_script branch yet, but feel free to do so if everything you need is merged into the master

**Pull request merged**